docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/PermutationFeatureImportance.cs

```diff
@@ -35,18 +40,29 @@ public static void Example()
             // Extract the predictor.
             var linearPredictor = model.LastTransformer;
 
-            // Compute the permutation metrics for the linear model using the normalized data.
-            var permutationMetrics = mlContext.Regression.PermutationFeatureImportance(
+            // Compute the permutation metrics for the linear model using the
+            // normalized data.
+            var permutationMetrics = mlContext.Regression
+                .PermutationFeatureImportance(
                 linearPredictor, transformedData, permutationCount: 30);
 
-            // Now let's look at which features are most important to the model overall.
-            // Get the feature indices sorted by their impact on RMSE.
-            var sortedIndices = permutationMetrics.Select((metrics, index) => new { index, metrics.RootMeanSquaredError})
-                .OrderByDescending(feature => Math.Abs(feature.RootMeanSquaredError.Mean))
+            // Now let's look at which features are most important to the model
+            // overall. Get the feature indices sorted by their impact on RMSE.
+            var sortedIndices = permutationMetrics
+                .Select((metrics, index) => new { index,
+                    metrics.RootMeanSquaredError})
+
+                .OrderByDescending(feature => Math.Abs(
+                    feature.RootMeanSquaredError.Mean))
+
```

Comment:
lines 54 and 57 are empty lines. the tabbing looks good but you don't need the extra lines.